packages/components/package.json

```diff
@@ -102,6 +102,7 @@
     "@types/react-router": "3.0.13",
     "@types/react-select": "1.0.57",
     "@types/react-test-renderer": "16.9.1",
+    "@welldone-software/why-did-you-render": "4.2.1",
```

Comment:
Why is this being added?

Reply:
Oops, it's not.  I just removed it.  Useful package though for optimizing renders.